vector/v.category/main.c

```diff
@@ -181,7 +212,45 @@ int main(int argc, char *argv[])
         break;
     }
 
+    /* read format */
+    switch (format_opt->answer[0]) {
+    case ('j'):
+        format = JSON;
+        break;
+    case ('s'):
+        format = SHELL;
+        break;
+    case ('p'):
+        format = PLAIN;
+        break;
+    }
+    if (shell->answer) {
```

Comment:
The shell option description should be probably changed to inform this flag is obsolete and will be removed in the future and then if selected, it should raise a warning about it being obsolete.